src/App/Action/IndexAction.php

```diff
@@ -23,6 +23,19 @@ public function __construct(TemplateRendererInterface $templateRenderer)
 
     public function __invoke(Request $request, Response $response, callable $next = null) : HtmlResponse
     {
-        return new HtmlResponse($this->templateRenderer->render('app::index'));
+        $currentDate = strtotime(date('U'));
+        $nextMeet    = strtotime('second wednesday of '.date('M'));
+        if ($currentDate >= $nextMeet) {
+            $nextMeet = strtotime('second wednesday of '.date('M', strtotime("+1 month")));
+        }
+
+        return new HtmlResponse(
+            $this->templateRenderer->render(
+                'app::index',
+                [
+                    'nextMeet' => date('l jS \of F Y', $nextMeet),
```

Comment:
Need some test cases to ensure this doesn't break, or switch to using the meetup service (or make a new `interface`).